lib/stream/xlsx/worksheet-writer.js

```diff
@@ -246,6 +252,7 @@ class WorksheetWriter {
     this._writeBackground();
     this._writeHeaderFooter();
     this._writeRowBreaks();
+    this._writeTables();
```

Comment:
Related to https://github.com/exceljs/exceljs/pull/2079 , the line this._writeTables(); needs to be after the legacyData otherwise the notes in a streamed table make the resulting file invalid in Excel